src/components/Article/Article.tsx

```diff
@@ -38,12 +51,28 @@ function renderCompetenceGoals(article, locale, isTopicArticle, subject) {
   return null;
 }
 
-const renderNotions = (article, locale) => {
-  const notions = article.concepts?.map(concept => {
+interface Props {
+  article: GQLArticle;
+  resourceType: string;
+  isTopicArticle: boolean;
+  children: React.ReactNode;
+  contentType: string;
+  label: string;
+  subject: GQLSubject;
+  locale: LocaleType;
+  isResourceArticle: boolean;
+  copyPageUrlLink: string;
+  printUrl: string;
+}
+
+const renderNotions = (article: GQLArticle, locale: LocaleType) => {
+  const notions = article.concepts?.map((concept: GQLConcept) => {
     const { content: text, copyright, subjectNames, visualElement } = concept;
-    const { creators: authors, license } = copyright;
+    const { creators: authors, license } = copyright!;
     return {
       ...concept,
+      id: concept.id?.toString()!,
+      title: concept.title!,
```

Comment:
Trengs denne når du spreader concept rett over?

Reply:
Ja, for Concept-typen har title som optional, mens det objektet som returneres krever at `title` er ikke optional